src/apps/menu/menu.tsx

```diff
@@ -157,89 +161,112 @@ const Menu: FC = () => {
-          </Link>
-        </div>
-        <div className="modal-profile__notifications">
-          {loansOverdue !== 0 && (
-            <MenuNotification
-              notificationNumber={loansOverdue}
-              notificationText={t("menuNotificationLoansExpiredText")}
-              notificationColor="danger"
-              notificationLink={menuNotificationLoansExpiredUrl}
-            />
-          )}
-          {loansSoonOverdue !== 0 && (
-            <MenuNotification
-              notificationNumber={loansSoonOverdue}
-              notificationText={t("menuNotificationLoansExpiringSoonText")}
-              notificationColor="warning"
-              notificationLink={menuNotificationLoansExpiringSoonUrl}
-            />
-          )}
-          {reservationsReadyForPickup !== 0 && (
-            <MenuNotification
-              notificationNumber={reservationsReadyForPickup}
-              notificationText={t("menuNotificationReadyForPickupText")}
-              notificationColor="info"
-              notificationLink={menuNotificationReadyForPickupUrl}
-            />
-          )}
-        </div>
-        <div className="modal-profile__container">
-          <div className="modal-profile__links">
-            <div className="link-filters">
-              <MenuNavigationList
-                menuNavigationData={menuNavigationData}
-                loansCount={loansCount}
-                reservationCount={reservationCount}
-                feeCount={feeCount}
+    <>
+      <button
+        className="header__menu-profile header__button"
+        type="button"
+        onClick={() => setShowMenu(true)}
+      >
+        <img src={profileIcon} alt="Profile" />
+      </button>
+      {showMenu && (
+        <div className="modal-backdrop">
+          {/* TODO: HANDLE AUTHENTICATION */}
+          {isLoggedIn && (
+            <div className="modal modal-show modal-profile modal-right">
+              <div
+                className="modal__screen-reader-description"
+                id="describemodal"
               />
-            </div>
-          </div>
-          <div className="modal-profile__btn-logout">
-            <Link href={menuLogOutUrl}>
               <button
                 type="button"
-                className="btn-primary btn-filled btn-medium arrow__hover--right-small undefined"
+                aria-describedby="describemodal"
+                className="btn-ui modal-btn-close"
+                aria-label="close modal"
+                onClick={closeLoanerMenu}
               >
-                {t("menuLogOutText")}
-                <div className="ml-16">
-                  <ArrowIcon />
-                </div>
+                <img src={CloseIcon} alt="close modal button" />
               </button>
-            </Link>
-          </div>
+              <div className="modal-header">
+                <div className="modal-header__avatar">
+                  <div className="avatar bg-global-secondary">
```

Comment:
Modal header is very generic. Would it be more precise to mention that is has something to do with the menu?